microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -22,89 +22,105 @@ ms.date: 05/04/2021
 
 
 <!-- ====================================================================== -->
-## Browse resources in the Network panel
+## Browse resources in the Network tool
 
 <!-- to do x, -->
+See [Log network activity](../network/index.md#log-network-activity) in the article  _Inspect network activity_ to learn how to see the network activity generated by the page resources.
 
-1. Open [Log network activity](../network/index.md#log-network-activity) (in the article  _Inspect network activity_) in a new tab or window.
+<!-- 1. Open [Log network activity](../network/index.md#log-network-activity) (in the article  _Inspect network activity_) in a new tab or window. --.
 
 <!-- 1. Do the steps in that section, and then continue below. -->
 
  <!-- open [link] in a new tab or window, and then continue below.. -->
 <!--why?-->
 
 <!-- need lead-in -->
+Show network activity for page resources:
 
 ![Page resources in the Network Log.](../media/resources-network-resources.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Browse resources by directory in the Sources tool
 
-To view the resources of a webpage organized by directory:
+You can use the **Sources** tool to see webpage resources organized by directory.
+
+To use the Sources tool:
 
 1. Open DevTools.
 
 1. Select the **Sources** tool, and then in the **Navigator** pane in the upper left, select the **Page** tab.
 
 1. Click the **More options** (...) button to the right of the **Page** tab, and then select **Group by folder**.
 
+   Open Sources tool:
+
    ![The Page tab in the Navigator pane of the Sources tool.](../media/resources-sources-page-empty.msft.png)
 
     Here is a breakdown of the non-obvious items in the previous figure.
 
     | Page item | Description |
-    |:--- |:--- |
+    |---|---|
     | `top` | The main document [browsing context](https://developer.mozilla.org/docs/Web/HTML/Element/iframe). |
-    | `airhorner.com` | The domain.  All resources nested under it come from that domain.  For example, the full URL of the `comlink.global.js` file is probably `https://airhorner.com/scripts/comlink.global.js`. |
-    | `scripts` | A directory. |
+    | `microsoftedge.github.io` | The domain.  All resources nested under it come from that domain.  For example, the full URL of the `buttons.js` file is [https://microsoftedge.github.io/Demos/devtools-a11y-testing/buttons.js](https://microsoftedge.github.io/Demos/devtools-a11y-testing/buttons.js). |
+    | `css`, `photos` | Directories |
     | `(index)` | The main HTML document. |
-    | `sw.js` | A service worker runtime context. |
+    | `buttons.js` | A script to handle button presses on the page. |
 
-1. Select a resource to view it in the **Editor**.
+1. Select a resource to view it in the **Editor**:
 
    ![View a file in the Editor.](../media/resources-sources-page-resource.msft.png)
 
```

Comment:
change from:
## Browse resources by filename in the Sources tool
to:
## Browse resource files sorted by filename in the Page tab of the Sources tool

Reply:
To check status, see:
Latest Files Changed view:
https://github.com/MicrosoftDocs/edge-developer/pull/2339/files
Rendered;
https://github.com/MicrosoftDocs/edge-developer/blob/user/v-danwesley/gs-view-page-resources/microsoft-edge/devtools-guide-chromium/resources/index.md

Reply:
Actual heading at the moment is:
Browse resources by filename in the Sources tool
which != the suggestion, of:
Browse resource files sorted by filename in the Page tab of the Sources tool

Reply:
applied locally